cmd/src/campaigns_new.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"text/template"
+
+	"github.com/pkg/errors"
+	"github.com/sourcegraph/src-cli/internal/campaigns"
+)
+
+func init() {
+	usage := `
+'src campaigns new' creates a new campaign spec YAML, prefilled with all
+required fields.
+
+Usage:
+
+    src campaigns new [-f FILE]
+
+Examples:
+
+
+    $ src campaigns new -f campaign.spec.yaml
+
+`
+
+	flagSet := flag.NewFlagSet("new", flag.ExitOnError)
+
+	var (
+		fileFlag = flagSet.String("f", "campaign.yaml", "The name of campaign spec file to create.")
+	)
+
+	handler := func(args []string) error {
+		if err := flagSet.Parse(args); err != nil {
+			return err
+		}
+
+		if _, err := os.Stat(*fileFlag); !os.IsNotExist(err) {
```

Comment:
nitpick: technically, it can also be other errors than file already exists

Reply:
Fixed!